### PR TITLE
fix(archive): Reset vertical scroll when navigating folders

### PR DESCRIPTION
**Pull request by mxiao6**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mxiao6 has signed the CLA. Thanks for the pull request!

File: test/integration/archive/ArchiveViewer.e2e.test.js [hunk 1]
```diff
@@ -53,4 +56,30 @@ describe('Archive Viewer', () => {
...
+        cy.getByTitle('Collection').within(() => {
+            cy.get('button').click();
+        });
+
+        cy.get('.ReactVirtualized__Table__Grid').then($tables => {
+            const table = $tables[0];
+            table.scrollTo(0, 50);
+
+            const tableTop = table.getBoundingClientRect().top;
+            cy.get('.ReactVirtualized__Table__row').then($rows => {
```
Comment (jstoffan) [inline test/integration/archive/ArchiveViewer.e2e.test.js]: Any way we can avoid using CSS classes here?

File: test/support/constants.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ Cypress.env({
     // NOTE: The token used is a readonly token accessing public data in a demo enterprise.
     ACCESS_TOKEN: 'S8wjvjOL9GEK5VtXsQNVMOwSrx1g55oC',
     FILE_ID_ARCHIVE: '589625563707',
+    FILE_ID_ARCHIVE_LONG: '611800378713',
```
Comment (jstoffan) [inline test/support/constants.js]: It looks like we have `FILE_ID_DOC_LARGE`. Can we rename to `FILE_ID_ARCHIVE_LARGE`, as well?

File: test/integration/archive/ArchiveViewer.e2e.test.js (hunk 1 above)
Comment (mxiao6) [reply test/integration/archive/ArchiveViewer.e2e.test.js]: Is using filename (like `cy.contains('File001.txt')`) better?

Comment (jstoffan) [reply test/integration/archive/ArchiveViewer.e2e.test.js]: I think so, yeah, since it's based on user-visible information. We do it in other tests, too.

**Pull request merged**